index.html

```diff
@@ -1,657 +1,1295 @@
                 </div>
-                <div class="btn-wrap">
-                    <div class="button">
-                        <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                            <span class="btn-text">Subscribe to Our Newsletter</span>
-                        </a>
+              </div>
+
+              <div class="container responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid button_block">
+                <div id="container-fcf0395128" class="cmp-container">
+                  <div  class="aem-Grid aem-Grid--12 aem-Grid--default--12 button-wrap">
+                    <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <a id="button-4baaf88684" class="cmp-button" href="/content/leaderimpact/lang/en2/stories.html">
+                        <span class="cmp-button__text">Subscribe to Our Newsletter</span>
+                      </a>
                     </div>
-                    <div class="button">
-                        <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                            <span class="btn-text">Join a Group</span>
-                        </a>
+                    <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <a id="button-1a8e1d6232" class="cmp-button" href="/content/leaderimpact/lang/en2/join-a-group.html">
+                        <span class="cmp-button__text">Join a Group</span>
+                      </a>
                     </div>
-                    <div class="button">
-                        <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                            <span class="btn-text">Start a Group</span>
-                        </a>
+                    <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <a id="button-465974c6ac" class="cmp-button" href="/content/leaderimpact/lang/en2/start-a-group.html">
+                        <span class="cmp-button__text">Start a Group</span>
+                      </a>
                     </div>
+                  </div>
                 </div>
+              </div>
             </div>
-        </section>
-        <section class="latest-post">
-            <div class="container">
-                <div class="sub-title">
-                    <div data-cmp-data-layer="" id="" class="cmp-title">
-                        <h6>Resources</h6>
-                    </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid latest-post aem-GridColumn aem-GridColumn--default--12 layout-grid">
+          <div id="latest-container" class="cmp-container">
+            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+              <div class="sub-title aem-GridColumn aem-GridColumn--default--12 layout-grid">

```

Comment:
This div has the `title` class on it. See [here](https://github.com/CruGlobal/leader-impact-review/blob/aem-restyle/aem/aem-home.html#L518)